services/notification/notifiers/mixins/message/sections.py

```diff
@@ -42,9 +42,11 @@ def get_section_class_from_layout_name(layout_name):
         return ImpactedEntrypointsSectionWriter
     if layout_name == "announcements":
         return AnnouncementSectionWriter
-    if layout_name == "header":
-        return HeaderSectionWriter
-    if layout_name == "newheader" or layout_name == "condensed_header":
+    if (
+        layout_name == "header"
```

Comment:
[very nit] `layout_name in ['header', 'newheader', 'condensed_header']` is more concise and should yield the same result.